auto3dseg/algorithm_templates/dints/scripts/algo.py

```diff
@@ -400,7 +416,7 @@ def train(self, train_params=None):
         # training
         dints_train_params = {}
         for k, v in params.items():
-            if k == "CUDA_VISIBLE_DEVICES":
+            if k not in allow_train_set:
```

Comment:
same here. we can add additional warning messages here if the given keys are non-existed.

Reply:
Addressed as above.